src/test/java/seedu/address/logic/commands/HistoryCommandTest.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.logic.commands;
+
+
+import org.junit.jupiter.api.Test;
+import seedu.address.logic.CommandList;
+import seedu.address.logic.CommandRemark;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.Assert.assertThrows;
+
+public class HistoryCommandTest {
+    @Test
+    public void history_command_test1() {
```

Comment:
Tests should be named according to this convention, i.e., methodName_kindOfInput_kindOfBehaviour. An example is `execute_editMemoUnfilteredList_success()`.